autoware_launch/config/perception/occupancy_grid_map/multi_lidar_pointcloud_based_occupancy_grid_map.param.yaml

```diff
@@ -12,6 +12,10 @@
         map_length_x: 150.0 # [m]
         map_length_y: 150.0 # [m]
 
+      # downsample input pointcloud
+      downsample_input_pointcloud: true
+      downsample_voxel_size: 0.125 # [m]
```

Comment:
```suggestion
      downsample_voxel_size: 0.25 # [m]
```
Why was the experiment done at 0.25, but now it is at 0.125?

Reply:
![image](https://github.com/autowarefoundation/autoware_launch/assets/20086766/1f59a61a-6390-4c05-92d2-9ab12031fdd9)